internal/static/cartogram2.js

```diff
@@ -1030,8 +1244,11 @@ class CartMap {
 
         }, this);        
 
+        this.getTotalValuesForVersion(new_sysname);
```

Comment:
Do you need these function calls to `getTotalValuesForVersion`, `getTotalAreaForVersion`, and `getConvenAllArea`?